web/src/app/browser/src/defaultBrowserRules.ts

```diff
@@ -38,7 +38,7 @@ export default class DefaultBrowserRules extends DefaultRules {
       const contextManager = this.contextManager;
       const activeElement = contextManager.activeTarget?.getElement();
       const nextElement = contextManager.page.findNeighboringInput(activeElement, back);
```

Comment:
The function can explicitly return `null`, yet we weren't null-guarding on the result.